tests/kernel/ib_tests.pm

```diff
@@ -42,19 +43,28 @@ sub check_dmesg {
     );
 }
 
+sub log_upload {
+    # just save the dmesg log
+    script_run("dmesg > /tmp/dmesg.txt");
+    upload_logs("/tmp/dmesg.txt");
+
+    # remove non-printable characters
+    script_run('tr -cd \'\11\12\15\40-\176\' < results/TEST-ib-test.xml > /tmp/results.xml');
+    parse_extra_log('XUnit', '/tmp/results.xml');
+
+    script_run("scp -o StrictHostKeyChecking=no root\@$slave:/tmp/dmesg.txt /tmp/dmesg_slave.txt");
```

Comment:
I always like to use StrictHostKeyChecking=no  together with UserKnownHostsFile=/dev/null